source/includes/siteurls-config/_20-create-and-update.md.erb

```diff
@@ -0,0 +1,47 @@
+## Create or Update siteUrls Config
+
+```shell
+$ curl 'https://ci.example.com/go/api/admin/config/server/site_urls' \
+      -u 'username:password' \
+      -H 'Accept: <%= data.apis.versions.siteurls_config %>' \
+      -X POST \
+      -H 'Content-Type: application/json' \
+      -d '{
+            "site_url" : "http://siteUrl.com",
+            "secure_site_url" : "https://secureSiteUrl.com"
+          }'
+```
+
+> The above command returns JSON structured like this:
+
+```http
+HTTP/1.1 200 OK
+Content-Type: <%= data.apis.versions.siteurls_config %>; charset=utf-8
+```
+
+```json
+{
+  "_links" : {
+    "doc" : {
+      "href" : "https://api.gocd.org/19.11.0/#create-site-url"
```

Comment:
`#create-or-update-siteurls-config` ?

Reply:
Will update this to top-level docs link i.e. `#siteurls-config` as all endpoints return top-level link only